source/backend/api/Services/AcquisitionFileService.cs

```diff
@@ -406,7 +412,32 @@ private void ValidateMinistryRegion(long acqFileId, short updatedRegion)
             short currentRegion = _acqFileRepository.GetRegion(acqFileId);
             if (currentRegion != updatedRegion)
             {
-                throw new BusinessRuleViolationException("The Ministry region has been changed, this will result in a change to the file's prefix. This requires user confirmation.");
+                throw new UserOverrideException("The Ministry region has been changed, this will result in a change to the file's prefix. This requires user confirmation.", "region_violation");
```

Comment:
This is similar to my updates for psp-5578, I'm happy to have your work merged first and then I will address the conflicts.